### PR TITLE
DrawNode::drawPie

### LINKED ISSUE #1447: DrawNode 2.0: Some new features
Feature 1
image

### PR DESCRIPTION
**Pull request by aismann**
Which branch your pull-request should merge into?

- `dev`: Current 2.x BugFixs or Features

## Describe your changes
see #1447 DrawNode::drawPie
see #1436 How to draw a sector-shaped area?

## Issue ticket number and link
#1447 

## Checklist before requesting a review
-  [x] I have performed a self-review of my code.
-  [x] If it is a core feature, I have added thorough tests.
-  [x] I have checked readme and add important infos to this PR (if it needed).
-  [x] I have added/adapted some tests too.

### CODE REVIEW

File: core/2d/DrawNode.h [hunk 1]
```diff
@@ -57,6 +57,19 @@ class PointArray;
 class AX_DLL DrawNode : public Node
 {
 public:
+
+    /** Different draw modus types.
+     *
+     *.
+     */
+    enum drawModus
```
Comment (halx99) [inline core/2d/DrawNode.h]: code style: `DrawModus` or `DRAW_MODUS`

**Comment by rh101** [comment]
Just a suggestion regarding the naming - please consider using "DrawMode" instead of "DrawModus", as it may be more appropriate given the context of what that enum is actually representing. 

@aismann If that enum is just for the drawing of a sector of a circle, then perhaps even call it "SectorDrawMode" (assuming it doesn't apply to any other DrawNode code).

**Comment by aismann** [comment]
> SectorDrawMode

At the moment 'enum DrawMode' can be used for "all" new drawing primitives like stars, gears, etc.

**Comment by aismann** [comment]
@halx99 
Can be merged now.

**Pull request merged**